tensorflow/python/kernel_tests/math_ops/argmax_op_test.py

```diff
@@ -96,6 +96,16 @@ def testFloat(self):
     self._testTieBreaking(np.float32)
     self._testDim(np.float32)
 
+  def testFloat16(self):
```

Comment:
Rather than duplicate the test logic, can you either loop over the dtypes or create a parameterized test?

Reply:
Thanks for the review @cantonios, updated the tests.